oas_docs/makefile

```diff
@@ -52,8 +57,9 @@ api-docs-overlay: ## Run spectral API docs linter on kibana.serverless.yaml
 	@npx bump overlay "output/kibana.tmp1.yaml" "overlays/alerting.overlays.yaml" > "output/kibana.tmp2.yaml"
 	@npx bump overlay "output/kibana.tmp2.yaml" "overlays/connectors.overlays.yaml" > "output/kibana.tmp3.yaml"
 	@npx bump overlay "output/kibana.tmp3.yaml" "overlays/kibana.overlays.shared.yaml" > "output/kibana.tmp4.yaml"
-	@npx @redocly/cli bundle output/kibana.serverless.tmp4.yaml --ext yaml -o output/kibana.serverless.new.yaml
-	@npx @redocly/cli bundle output/kibana.tmp4.yaml --ext yaml -o output/kibana.new.yaml
+	@npx @redocly/cli bundle output/kibana.serverless.tmp4.yaml --ext yaml -o output/kibana.serverless.yaml
+	@npx @redocly/cli bundle output/kibana.tmp4.yaml --ext yaml -o output/kibana.yaml
+	@sed -i '' -e 's/; Elastic-Api-Version=[0-9]\{4\}-[0-9]\{2\}-[0-9]\{2\}//g' output/kibana.yaml
```

Comment:
@lukeelmers this is where we remove the `Elastic-Api-Version` from the stateful bundles. For now I think this should be OK as, in future, we might choose to introduce them there more formally.

Reply:
thanks! does it need to be there in serverless too... it feels like we could remove it in both places, right? (since it looks like this is the one that's in the `content-type` which afaik nobody needs to actually include)

Reply:
my concern for stateful is that it will be confusing as we have not formally/publicly explained how the dual date/stack versioning works there.

my concern for serverless is that it is just redundant and also a bit confusing since content-type does not dictate api version (unless I missed something)

Reply:
> my concern for serverless is that it is just redundant and also a bit confusing since content-type does not dictate api version (unless I missed something)

My understanding is it is intended to be used for grouping our serverless versions. Until that strategy changes from docs side I don't think we should remove it for now.

Reply:
Relates to https://github.com/elastic/docs-content/issues/142 As soon as we start having multiple valid in-support elastic-api-version values in stateful or serverless, we'll need to be able to differentiate their impact on the requests and responses.

Reply:
> ... we have not formally/publicly explained how the dual date/stack versioning works there.

Whenever we're ready to document that, it can be added to the API docs in a topic similar to https://www.elastic.co/docs/api/doc/kibana/topic/topic-kibana-spaces, which we add via  https://github.com/elastic/kibana/blob/d601e23c40be2b83c75be23c921ac42a39e034dd/oas_docs/overlays/kibana.overlays.yaml#L11